packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+    border-color: var(--terra-slider-thumb-hover-border-color, #0065a3);
+  }
+
+  .input-range:active::-webkit-slider-thumb {
+    background: var(--terra-slider-thumb-active-background, #007cc3);
+    border-color: var(--terra-slider-thumb-active-border-color, #0065a3);
+  }
+
+  .input-range:hover::-webkit-slider-runnable-track {
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range-hover, #0065a3) 0%, var(--terra-slider-track-selected-range-hover, #0065a3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range-hover, #bcbfc0) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range-hover, #bcbfc0) 100%);
+  }
+
+  .input-range:disabled::-webkit-slider-runnable-track {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-disabled-selected-range, #007cc3) 0%, var(--terra-slider-track-disabled-selected-range, #007cc3)  50%, var(--terra-slider-track-disabled-unselected-range, #d3d4d5)  50%, var(--terra-slider-track-disabled-unselected-range, #d3d4d5) 100%);
+    
+  }
+
+  .input-range:disabled::-webkit-slider-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: #007cc3;
+  }
+
+  /* For IE/Edge */
+  .input-range::-ms-track {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range, #007cc3) 0%, var(--terra-slider-track-selected-range, #007cc3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) 100%);
+    border-radius: 3px;
+    height: 6px;
+    width: 6px;
+  }
+
+  .input-range::-ms-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: var(--terra-slider-thumb-background, #007cc3);
+    border-color: var(--terra-slider-thumb-border-color, #0065a3);
+    border-radius: 50%;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.25);
+    cursor: pointer;
+    height: 24px;
+    margin-top: -8px;
+    width: 24px;
+  }
+
+  .input-range:focus::-ms-thumb {
+    border-color: var(--terra-slider-thumb-focus-border-color, #007cc3);
+    border-width: 1px;
+    outline: var(--terra-slider-thumb-focus-outline, 2px dashed #000);
+    outline-offset: 1px;
+  }
+
+  .input-range:hover::-ms-thumb {
+    background: var(--terra-slider-thumb-hover-background, #0065a3);
+  }
+
+  .input-range:active::-ms-thumb {
+    background: var(--terra-slider-thumb-active-background, #007cc3);
+  }
```

Comment:
The colors for focus and active are the same.  You should be able to combine those styles in your stylesheet.

Reply:
Updated these colors. They are different now.